### PR TITLE
[@kadena/transfer] New Layout

### PR DESCRIPTION
**Pull request by rezaerami**

### CODE REVIEW

File: packages/apps/transfer/src/utils/styles.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  if (typeof prop === 'number') {
+    return {
+      width: getPropInPercentage(prop),
+    };
+  }
+
+  const { size, span, offset, pull, push, hidden } = prop;
+  const styles: IGridColMediaStyles = {};
+
+  if(size)  styles.width = getPropInPercentage(size).toString();
```
Comment (rezaerami) [inline packages/apps/transfer/src/utils/styles.ts:24]: needs to be done like this to make sure we don't have undefined properties in the result

File: packages/apps/transfer/src/utils/styles.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  if(span)  styles.flex = `0 0 ${getPropInPercentage(span)}`;
+  if(offset)  styles.marginLeft = getPropInPercentage(offset).toString();
+  if(push)  styles.marginRight = getPropInPercentage(push).toString();
+  if(pull)  styles.order = pull.toString();
+  if(hidden !== undefined)  styles.display = hidden ? 'none' : "unset";
```
Comment (rezaerami) [inline packages/apps/transfer/src/utils/styles.ts:29]: only if hidden is passed id needs to check the value!

File: packages/apps/transfer/src/components/Global/Grid/GridCol.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  sm?: IGridColMediaType,
+  md?: IGridColMediaType,
+  lg?: IGridColMediaType,
+  xl?: IGridColMediaType,
+  xxl?: IGridColMediaType,
+  className?: string
+  children: ReactNode
+}
+
+const GridCol: FC<IGridRowProps> = ({children, className, xs = {}, sm={}, md={}, lg={}, xl={}, xxl = {}}) => {
```
Comment (rezaerami) [inline packages/apps/transfer/src/components/Global/Grid/GridCol.tsx:19]: can't make it dynamic cause it'll be a runtime type checker so the build will fail

File: packages/apps/transfer/package.json [hunk 4]
```diff
@@ -21,9 +21,11 @@
     "@kadena/chainweb-node-client": "workspace:*",
     "@kadena/client": "workspace:*",
     "@kadena/cryptography-utils": "workspace:*",
+    "@kadena/kadena-docs": "workspace:*",
     "@kadena/pactjs": "workspace:*",
     "@kadena/pactjs-cli": "workspace:*",
     "@kadena/react-components": "workspace:*",
+    "@kadena/react-ui": "workspace:*",
     "@radix-ui/colors": "~0.1.8",
     "@stitches/react": "1.3.1-1",
```
Comment (MRVDH) [inline packages/apps/transfer/package.json:30]: Can't stitches be removed with the new layout?

File: packages/apps/transfer/src/components/Common/Layout/index.tsx [hunk 5]
```diff
@@ -1,14 +1,52 @@
+import {Header} from "./partials";
+
 import { StyledLayout } from '@/components/Common/Layout/styles';
+import {WalletConnectButton} from "@/components/Global";
+import {Logo} from "@/resources/svg/generated";
+import useTranslation from "next-translate/useTranslation";
 import React, { type ReactNode, FC } from 'react';
+import routes from "../../../constants/routes";
```
Comment (MRVDH) [inline packages/apps/transfer/src/components/Common/Layout/index.tsx]: absolute routes

File: packages/apps/transfer/src/components/Common/Layout/index.tsx [hunk 6]
```diff
@@ -1,14 +1,52 @@
+import {Header} from "./partials";
+
 import { StyledLayout } from '@/components/Common/Layout/styles';
+import {WalletConnectButton} from "@/components/Global";
```
Comment (MRVDH) [inline packages/apps/transfer/src/components/Common/Layout/index.tsx]: spacing should be fixed by the linter probably

File: packages/apps/transfer/package.json (hunk 4 above)
Comment (rezaerami) [reply packages/apps/transfer/package.json:30]: nope, we use old components also, until we remove all dependencies to old react components we need to keep them

File: packages/apps/transfer/package.json [hunk 7]
```diff
@@ -21,9 +21,11 @@
     "@kadena/chainweb-node-client": "workspace:*",
     "@kadena/client": "workspace:*",
     "@kadena/cryptography-utils": "workspace:*",
+    "@kadena/kadena-docs": "workspace:*",
```
Comment (webpro) [inline packages/apps/transfer/package.json:24]: Do we need/use this dependency (app in app)?

**Review by webpro — COMMENTED** [review]
Please run `rush[x] format` on updated packages.

File: packages/apps/transfer/package.json (hunk 7 above)
Comment (rezaerami) [reply packages/apps/transfer/package.json:24]: nope, i was trying to get docs component, thanks for reminding me, removed

**Review by alber70g — APPROVED** [review]

**Review by MRVDH — APPROVED** [review]

**Pull request merged**